### ISSUE TITLE
Including aeroo

### ISSUE THREAD

**Issue by eLBati** [OCA/reporting-engine#1]
Following up the discussions at https://github.com/OCA/hr/pull/17#issuecomment-55031370 and [odoo ML](https://www.odoo.com/groups/community-59/community-9023238), I open this issue to decide whether to include [aeroo modules](https://launchpad.net/aeroo) under the [OCA](http://odoo-community.org/) umbrella, or to move modules like [hr_payroll_register_report](https://github.com/OCA/hr/blob/master/hr_payroll_register_report/__openerp__.py) outside of the OCA.

**Comment by pedrobaeza** [comment]
I think Aeroo can be included into OCA if @sraps is OK about that and signs the CLA. Aeroo covers one need that none of the current reporting engines do: allow fast design by non-tech user. For me, it has served as a cheap way of having custom design reports in OpenERP, because they have it on DOC or XLS.

**Comment by jjscarafia** [comment]
- 1 for aeroo included into OCA

_Ing. Juan José Scarafía_
(+54 9 341)153 278039
skype: jjscarafia
twitter: @jjscarafia https://twitter.com/jjscarafia
github: @jjscarafia https://github.com/jjscarafia
www.ingadhoc.com

2014-09-12 11:10 GMT-03:00 Pedro M. Baeza notifications@github.com:

**Comment by sraps** [comment]
Well in practice I have several questions and they are not directly Aeroo related, for instance I have other projects as well, so as other's do as well. Passing away just one project without any particular reason, solved nothing. :
1. What in practice Aeroo (read any project) could give to OCA?
2. What in practice OCA can give to Aeroo (read any project)?
3. What in practice and by whom is being done to finance the development, communication, promotion of the project....?
4. How these efforts would be better than now, in particular?
5. What is the OCA role and goal in maintaining it's own Odoo branch?
6. Does OCA differentiate between project and product?
7. How does OCA sees itself to solving main problem of FOSS, creating the product then monetization of it? Producing +/- quality code in great amounts is allready solved, isn't it?

IMHO the last one is the most important question of all. At the moment I have no answers to a single one of them.

**Comment by dreispt** [comment]
Great questions, and the answers should be compiled in a FAQ next to a "How to contribute" guideline.

**Comment by hbrunn** [comment]
1. The code + permissions as outlined here: http://odoo-community.org/page/website.cla
2. Faster bug fixes, more input, more reviews, less maintenance on your own because the burden is shared
3. Nothing at the moment. It's a non-profit organization that members participate in voluntarily
4. Hopefully there are less never merged fixes than here: https://code.launchpad.net/aeroo/+activereviews, some of them very crucial ones. I must admit that I currently don't even bother to PR into your branch because of previous experience, and use this one for 8.0 improvements: https://github.com/jamotion/aeroo
5. The original idea is outlined here: https://lists.launchpad.net/openerp-community/msg02197.html tldr: Bug fixes only, support end of live time versions
6. can't answer that
7. same here

About 6 and 7: I personally don't think the OCA should be concerned with that, that's up to the businesses themselves. 1-5 tell you why it's still a good idea to join forces.

**Comment by sraps** [comment]
In general practice things happen like this if everything is ovned and managed by general public without anyone specific in charge:
- nothing is ever built from scratch by general public - every effort is being started by someone specific;
- nothing is ever developed to a higher level by general public - every effort is being executed by someone specific;
- things can be financed by general public - in fact many of great efforts are being financed by public;
- in a long term patch design is never sustainable - someone have to feel responsible for that, if noone does feel responsible noone in particular ever does care;
- problem with FOSS and general public is that everyone takes everything for granted, that's is so humane, one takes the branch where someone have done the job beforehand;
- general implementer are more likely to pay for total solution with FOSS, 0.001% wants to finance backoffice tools that made it possible;

That is why many of FOSS projects are in essence geek tools, workable still are not turnkey solutions. In my oppinion OCA should focus on maintaining and developing environment of FOSS ERP systems (which are totally different business model than mugware) so that every player feels comfortable, not only the central or end users.

Look at any other associations out there, most of them fight for better environment for those involved, none of them fight for better product. In contrast OCA stands for perfectioning product (introduce standards) and trying to push costs down (which is already not so high). Do I (I am speaking for myself) need this? No, I do not put anything of that in a first place.

At the moment problem is dog-eat-dog environment while we are supposed to work in technolgy business, but in fact are selling man-hours at lower possible cost. That is why most implementers of Odoo execute policy - rapid deployment (stated 3 months per project !!!), and run. Of course they are fixing some bugs in a way, but 1/1000 gives anything back, not even the ideas or technologies. Only few of the developers "eat their own dogfood".

I see no particular practical reason passing Aeroo to OCA:
- I see no reason it'd help me develop Aeroo;
- it see no way it'd help me or make possible developing new projects (efforts shall bring options, not more work, just to pay bills);
- it even are not capable of incepting and developing new projects from the ground on public basis, as it's not OCA purpose at the moment;

Concluding, I hope, I could push the movement towards solving real problems - selling the technologies, not patch-man-hours on other's developments, so that everyone involved benefits from the technology not only salesman or end users. And the reward would be really mature product not the patch blanket as it's now.

https://www.google.com/search?q=patch+blanket&tbm=isch&tbo=u&source=univ&sa=X&ei=iIIaVPrqF6WGywOKkYLoAg&ved=0CBwQsAQ&biw=1920&bih=959

**Comment by sraps** [comment]
http://odoo-community.org/page/website.faq

Does the Association need money?
The association is now soliciting organizational sponsorships. You can make a donation here. However, contributions of time for committee work, development, documentation, testing, user support, and advocacy are the preferred forms of contribution from individuals.

In essence this means:
1. no we don't need money;
2. we're fed up with the daily support and bugfixing and testing and we welcome volunteers (!!!), no skills required;
3. we need volunteers for user support - it means, we do not charge money for user support. Users are not our source of funding.

Sorry for my sarcasm.

**Comment by rvalyi** [comment]
@sraps, I think there is a point a missing point: without OCA, without an organized community, I'm pretty sure things can become pretty worse quite soon (brutal re-licensing, surprise roadmap, obsolescence by design etc...).

On the other hand, I also think that OCA will need to provide a fair visibility to module authors. Not unleashed advertisement but serious yet explicit visibility of the work done, possibly including HTML links. By doing so, implementation businesses could certainly get a benefit of image, specially in the high end implementations, that could sometimes replace in part marketing expense of any kind. That is you would get your business value. In fact, if on the contrary OCA dilute authors visibility and incentive users to think it's all for granted, then yes, OCA would instead destroy business value.

This isn't sorted out, but I personally wish we deal with these topics with the right amount of balance. May be some big guys of OCA are big enough to afford this to be just a give away. But we should not forget that if OCA goes with no business model at all, it will not attract many guys like you and it will not resist long against the power of money that will make whole different plans for the product.

my 2 cts

**Comment by pedrobaeza** [comment]
@sraps, are you aware that we don't need your permission to start an OCA fork of Aeroo (obviously respecting work attribution)? And we will do it if you don't want to join efforts, because as you know this tool is very useful, but I and a lot of people like me don't want to contribute to a personal branch having the possibility to make it under the umbrella of an organization that has also some quality tools and a critical mass that assures the continuity of the project (which you don't).

Sorry if I sound rude, but this is my opinion after collaborating with pre-OCA and now OCA for about 2 years.

Regards.

**Comment by sraps** [comment]
@rvalyi 
- Brutal re-licensing have happened before, we've all faced it;
- Surprise roadmap - focus and scope of Odoo have changed since v8;
- Obsolescence, with 1.5 y (which is nonsense by design) of life-cycle we are running from somebody I guess;
- totally agree on the rest;
- I would add project high jacking.

@pedrobaeza 
- are you aware that we don't need your permission to start an OCA fork of Aeroo (obviously respecting work attribution)?

Yes, I am, and that is best and still the worst idea being done every day. There is a huge difference in forking for your personal (in-company) use and forking well promoted software out of interest of polishing one's own image, like you are offering with Aeroo and OCA. If Aeroo is forked by OCA, be aware - at the moment OCA have done nothing so that Aeroo could be further developed, not to mention the efforts invested to promote Aeroo facing Odoo's constant ignorance for the REAL community investing big money in quality software.
- And we will do it if you don't want to join efforts, because as you know this tool is very useful, but I and a lot of people like me don't want to contribute to a personal branch having the possibility to make it under the umbrella of an organization that has also some quality tools and a critical mass that assures the continuity of the project (which you don't).

I guess OCA shall prove that critical mass, otherwise it looks more like aggressive threatening. I see just OCA saying "pass up or die". Let's face reality, OCA needs projects like Aeroo, the same as Aeroo needs REAL community involvement (not the show of public power) and financing if it's a community project for the community. Otherwise it's just a patchware and will die.

I'll make it clear, Aeroo can and shall be used by general public, but it is neither the result of it, nor developed or promoted by charitable efforts - moreover, ERPs are not made by charities.

If OCA does not find a way to enforce win-win situation, I see it as some repository for easier and ignorant use by the end no-customer. So that some bigshot can save $$$ not buying expensive proprietary ERP for their multi million enterprise. Yes every enterprises really using ERPs are spending at least hundreds of thousands on software, now it is spent mainly on the software it can not take for granted, for free.

## I need association to change this de-facto rule, not to take over Aeroo.

**Comment by StefanRijnhart** [comment]
Well, that escalated quickly. To get back to the original question: should OCA adopt the Aeroo project? One of the principles of the OCA is to prevent double work. So if Alistek wants to keep maintaining Aeroo by themselves, OCA should not interfere as this would lead to double work. But as with Odoo itself, community contributed bugfixes for sometimes very serious problems have been left for too long on the Aeroo project. So although I'd rather work with Alistek on an OCA Aeroo project, if that is not possible then my suggestion would be to start an Aeroo backports project, along the same principles as the Odoo backports.

**Comment by sraps** [comment]
@StefanRijnhart Please read the @rvalyi comments.

OCA does not fix problems of Aeroo's further development. I admit that there is a space for improvement in Aeroo maintenance, but this is due to the fact that there is mass of users of Aeroo and tiny fraction of contributors of Aeroo. And now all the user and implementer mass acknowledges using Odoo, none of them even bother being aware using Aeroo for generating their daily invoices. OCA does not even put this problem in their scope.

# This sounds like - give me engine for free, I swear to maintain it well, but maintenance is just part of the life-cycle, you know. Passing Aeroo to OCA saves the day for another ignorant user.

**Comment by sraps** [comment]
Moreover, there's not even clear who's in charge of OCA, is it a board or a single executive? Somebody responsible, tell me if OCA even wants to maintain Aeroo, dedicate some resources to it? Not even clear if it has such resources - just to maintain.

# It is wrong by design: "Pass over and we'll figure out what to do then!"

At the moment I hear rather aggressive "pass up or die", nobody from OCA board have even bothered contacting me directly and discuss the offer, just some talks in github issues. So, unless something changes, I consider, it's just a bad joke.

**Comment by pedrobaeza** [comment]
@sraps, there is a board in OCA and I suppose they will comment about, but I as a contributor think: I'm not going to contribute to your project where no rules are defined, no quality tools and so on. That's why I'm saying I'm going to fork anyway and see if OCA accepts my "contribution". If not, we continue as is. If yes, people (a lot or very few), will choose surely OCA fork, not your repository, to contribute. Also OCB mechanism, as @StefanRijnhart says, can be another method of forking. 

I want also a health ecosystem where all the involved people (integrators, developers, Odoo S. A.) can have a win-win situation, but this is another discussion that we already have in other projects. For instance, @ajuaristi and me are doing a big MRP project. If you pretend to "block" Aeroo entrance in OCA for this discussion, I think this is the wrong way.

Regards.

Regards.

**Comment by sraps** [comment]
@pedrobaeza 
At the moment I am a developer of Aeroo Reports and I am not implementer of it in any Odoo v8.
- For instance, @ajuaristi and me are doing a big MRP project.

You are not telling me no news. Many people benefit from Aeroo Reports still few of them cared to give back just a credit that they are using Aeroo Reports.

At the moment OCA is a maintainer of their own branch of Odoo, and their own branches of bunch of other software. So what? How does it solves the daily problems of OEM developers? Do they coordinate any new development, that everybody would benefit?

# So sorry, but OCA solves problems for end-user, not the developer. They get better software at virtually no cost.

Please, why do I, as a developer, need OCA?

**Comment by rvalyi** [comment]
@sraps,

again, when you say "OCA does not fix problems of Aeroo's further development", IMHO you see the half empty glass. I rather see the half full glass with the fact that OCA can prevent things to be much worse for everybody, including you and Aeroo. And this alone is IMHO a good reason to join forces and leave the FOSS challenges for another round and work them on a company basis meanwhile. Come on, do you think for a sec these pricing things will lead to anything sustainable? What kind of plan B can you imagine after plan A fail? Not so nice, right, so better be pragmatic right now.

**Comment by sraps** [comment]
@rvalyi 
Being pragmatic. While OCA can not show any movement towards funding and promoting, which is just a part of lifecycle, it's just another round of bureaucracy. I even do not see any committee for managing the projects, so it's again on our behalf, right? How does OCA help me maintain it?

When there none responsible, everything just slowly dies. There's not even any idea how to finance the work involved.

**Comment by rvalyi** [comment]
@sraps
today running after Aeroo fixes already costs everybody some time among the OCA active members. They simply make the guess that just like with the other projects, that cost will be less if there is a curated collective review. Even if half of the reviewer should once spend a review on it this will certainly be a win. That win can also certainly be yours. It won't help you so much get more money, but it will help you win time maintaining it so you could use your precious time doing something more interesting for your business. Well at least this is how I see it.

OCA does bring some burbureaucracy, that bureaucracy may certainly be too much for the little module your customer need next week. But for a large project such as Aeroo, in my opinion it's largely balanced by the benefits. I was a bit skeptical too at the beginning, but after some practice I certainly see positive results were we did it.

**Comment by sraps** [comment]
http://odoo-community.org/page/website.websiteOCA_Benefices

# Improve your visibility

How is the visibility improved for any OCA umbrella project? Please name example. At the moment I see no credit of any OEM, other than Odoo itself in the OCA name. It's just one way entry for the OEM, just like with the logo there are no-name ants, not even worth mentioning.

# Get yourself known

I see no option of even steering my own project. Read above...

# Go international

Aeroo is already international, it has users in any continent, but Antarctica.

# Enhance your quality

Please name example.

# Reduce your costs

Ok, it may (potentially) reduce costs of maintenance.

# Come by legitimacy and sponsorship

# Acquire funding support

How? Please name example. How money travel to the particular people, is there any standard scheme?

**Comment by dreispt** [comment]
@sraps Are you sure about Antarctica? :smile: 

Seriously, I feel that the discussion has not evolved from Stephan's post onward.
Adding a project to OCA does have it's trade-offs, and no one can be pushed into that.

It's clear that @sraps doesn't see a benefit in that, and that's OK.
It won't be in OCA, it's still open source, and people can work with it (and it's authors) as they do with any other piece of source code on the web.

So, I think a conclusion has been reached, let's move on.

**Comment by pedrobaeza** [comment]
Yeah, I already have the same feeling a few post ago, so I'll do a PR to reporting-engine project for Aeroo in the next days.

Regards.

**Comment by eLBati** [comment]
> Please, why do I, as a developer, need OCA?

@sraps because you get better quality software.
If you need examples, take a tour of the open PRs of a random OCA project like [server-tools](https://github.com/OCA/server-tools/pulls) or [sale-workflow](https://github.com/OCA/sale-workflow/pulls). You will see several odoo experts cooperating to improve the software quality.

> This sounds like - give me engine for free, I swear to maintain it well, but maintenance is just part of the life-cycle, you know

And your words sound to me like "I want to remain the maintainer of aeroo so that people will contact me when they need support"

**Comment by sraps** [comment]
Well, in essence, let's move to the time where there were no OCA. I contacted to some of then OpenERP community members with similar idea of creating something like OCA. But my primary scope and idea was, finding solution to the problems, practically defining environment around FOSS ERP product and companies working with it. The other thing was to find technical solutions like performance and floating point like problems, in which Odoo SA itself is not interested. Maintenance was the least of the problems.

At that moment I was even willing to move some of our projects under incepted association (Alistek has number of projects even lot bigger in codebase than Aeroo is, most of them are lacking of funding, not the hands doing a maintenance), mainly to promote new association. Ok, closer to the idea, scope was solving the problems, not just producing "solutions" to non-existent problems. The idea did not catch up.

Two/three years later, at the moment, I see Odoo's 2nd version that is announced as disruptive, but actually never released - v7 & v8., if we compare to v6.1. Real problems never solved.

Lot of crucial components just abandoned, like Application Client , lot of efforts spent on highly regulated and localized features like POS, which will never be an industry standard, because of wrong technical solutions chosen (I have been there) and highly regulated environment. Lot of efforts spent on making Odoo a web portal directly serving websites, none cares for the security concerns, etc.

Too little spent on fixing things like floating point accuracy, accounting &, stock performance. Virtually creating a product for tiny enterprises cloud hosting.

I see no stance of OCA, in these questions, at all. So what is then a point of passing Aeroo to OCA, just to continue maintaining my own project? It saves the day to someone out there, not for me. Maintenance costs are contributing to TCO, if someone has too high costs of maintaining Aeroo Reports, why then no company contacts Alistek for maintenance contract? So I guess all that are just speculations.

# I am very interested in answers to these questions, because they are essential, if somebody from OCA could answer, I will move Aeroo under OCA.

**Comment by sraps** [comment]
@eLBati 
- And your words sound to me like "I want to remain the maintainer of aeroo so that people will contact me when they need support"

Might be so, that is another point. Because you can not contact anybody if there's even no credit who's who for the project.

@pedrobaeza 
Let's face it, maintenance problems are mainly for those who are not technically oriented, in other words, end users. So if end-user has problem with maintenance, he contacts the developer for a maintenance contract, right? Those who knows what a development/maintenance mean, have something different daily agenda. I do not need PR, I want some improvement if moving to OCA. But I do not need improvement in end-user's TCO reduction, if they are not paying a single $ for that. It does not matter if it is Alistek or OCA.

If it's just better collaboration, Github is great tool for it.

**Comment by sraps** [comment]
Anyway there's another reporting tool on the block.

https://twitter.com/RohanNayani/status/512542049900318720/photo/1

**Comment by eLBati** [comment]
On 09/18/2014 03:50 PM, sraps wrote:

> @eLBati https://github.com/eLBati
> - And your words sound to me like "I want to remain the maintainer
>   of aeroo so that people will contact me when they need support"
> 
> Might be so, that is another point. Because you can not contact 
> anybody if there's even no credit who's who for the project.

Your name and URL would obviously remain in the author and website 
fields of the modules.

**Comment by pedrobaeza** [comment]
@sraps You can continue making philosophy about the open source ecosystem in general and Odoo one in particular, but this is a real question between developers and integrators, not for final users. I want this tool to be the best - technically and functionally speaking -, and I will collaborate for it, but not for your personal interest in monetizing this. Your credits will be still there, of course, for any customer (mine or from other) to contact you for what they want, and I also thank you to have created this tool, but don't try to retain the project in your hands for this purpose. You don't have any tool to automate and assure code quality, and your criteria to merge a contribution can be arbitrary, which it's not under OCA umbrella. 

My partner Ana Juaristi (who worked with me in OdooMRP project) has contributed already in the name of both for your still partial migration to v8, so it's not a question of getting things free. 

If you continue thinking in that closed way, on the "distribution channel" and on version progress, maybe you will not be the right person for the migration to v9 or v10, and someone arises to this figure and OCA (or someone else) want to fund that migration.

Regards.

**Comment by elicoidal** [comment]
Some points I would like to highlight here:
- OCA is a young association and not yet fully organized for founding 
  and stable organization. Nevertheless momentum is here and we are 
  getting more and more projects, more and more members and so far we 
  gathering strength more than splitting them
- OCA is organizing numerous repositories and 2 main projects 
  (MRP+Accounting for v9) and maintaining other deprecated features (OCB), 
  in order to give stability on long run for essential features (such as 
  for example the excellent Aeroo project)
- Nevetheless, OCA is a voluntary association. Is free to participate 
  whoever wants to. if @sraps is not willing to, this is completely fine 
  and we are nobody to judge this position.
- OCA is a community association which accepts projects around Odoo. A 
  fork of Aeroo as a backport is welcome if a group of people maintains it 
  and the copyright is respected.

In short, joining OCA allows you to:
- Gather strength and make sure we do not depend on one individual or 
  organization.
- Make sure features are maintained on longer run
- Quality code review
- Get features that would not be accepted in the core

@sraps,
If you feel that our objectives can be common, you are welcome to join 
and we will do our best to properly maintain together your work in the 

## long run

Eric CAUDAL

Eric Caudal
/Member of the board/
*Odoo Community Association  http://community.odoo.com
*

**Comment by sraps** [comment]
@pedrobaeza 
There are number of people I have heard of only when there were another fork of Aeroo Reports emerged, never before and never after. People distributing somehow arguable communication for better quality, code reviews and super cool features. Where are they now, who knows. I have seen the same situation with most of our projects released as FOSS, people fork rename, distribute false communication about improvements and authorship. Nevertheless Aeroo is the only "fork" that survived since TinyERP v 4.2 times, and this aggressive movement seems like is your's and nobody's else opinion.
Aeroo indeed is half ported to v8, because there is still no v8 - every day there's some sporadic changes in a so called disruptive super version of Odoo, long since released, but still green. I you have not noticed all the porting efforts of Aeroo for previous OpenERP versions, have been done on almost ready releases, because only madman would implement such alpha, proof-of-concept versions in a working environment, which is v8 at the moment. The same policy followed Alistek's implementations, we've always done implementations on previous versions, just because of too naive release policy of OpenERP, sorry but it's true. When there's announced new super verssion, the "obsolete" one is almost production safe. This time it's not an exception, Aeroo is ported at the moment when there is fewer core changes in Odoo, and posted issues on core relating Aeroo are closed. Whether do you like it or not.
Regarding Aeroo and Odoo v9, I do not know if there will still be Aeroo and Odoo would not be transformed into facebook. We'll see that.

@elicoidal
First of all, thank you for your involvement. At the moment I see OCA's movement towards improving other's software quality like Odoo backports, in other terms improve that the other's have failed doing, mostly in terms of funding. I think if OpenERP SA, just like other projects, would have enough funding, they'd have already created best ERP ever.

I would prefer, if OCA would focus on improving overall collaboration quality, not only the technical quality. If this would become OCA's goal I would be happy to join it and help establish better environment for Odoo related companies. Now it's just a way someone saves customer's money.

**Comment by elicoidal** [comment]
@sraps,
OCA motive is definitively to move beyond simple code review and 
repository management but we are currently stuck in this task as it is 
high workload and essential to start other phases later.
We are thinking about functional documentation, apps visibility, 
crowdfunding, sponsor management, project in direct cooperation with 
OpenERP SA etc.
After the code repositories are stabilized, we will start considering 
actions in this direction but we cannot extinguish too many fires at the 
same time.
Eric CAUDAL

Eric Caudal

**Comment by gurneyalex** [comment]
AFAICT we have no signed CLA from @sraps. As a result, report-aeroo cannot be added in its current state to OCA.

**Comment by sraps** [comment]
As I have said, OCA neither solves any of support problems, nor provides any visibility to the project or Alistek. It's just a way somebody can get more free beer, though (probably) at a better quality. OCA is what it stands for - effectively promoting Odoo, not Aeroo or any project OCA hosts. As well, I envision Aeroo may be ported to F3 or Tryton or any other business related application.

So as long as OCA is about promotion of Odoo and single stop agency for free beer seekers, not the organization that organizes efforts and help companies/individuals standing behind the software, I see no purpose on giving up the freedoms and options of being core developer and brand holder.

Also I invite everybody using Aeroo Reports participating in making Aeroo better and sharing code, experience and expertise. What do you think if Aeroo related modules could be hosted as a separate projects along Aeroo Reports?

So effectively, there are two options:
1. Owners host the Aeroo related projects on their own;
2. host along Aeroo Reports.

My idea is that projects related to the technology in itself, call it generic functional  mlodules - not the report modules, could be hosted along Aeroo Reports. Modules encapsulating particular reports should be hosted along core module that provides the functionality, the report has been developed for.

**Comment by StefanRijnhart** [comment]
Obviously I don't agree with some of your other words on the OCA but I do like your characterization of it as "more free beer, though (probably) at a better quality"! And this is indeed accomplished by a collaboration of people who are more interested in improving the quality of their services by sharing their effort rather than promoting their own organization or brand names.

I would suggest that @gurneyalex remark finalizes this discussion and this issue can be closed.

**Comment by sraps** [comment]
I fully admit that "And this is indeed accomplished by a collaboration of people who are more interested in improving the quality of their services by sharing their effort rather than promoting their own organization or brand names."

@StefanRijnhart which is one of the problems behind ecosystem - lack of clear vision of business, the motive to make better product. Far too many people are doing this for fun, not for profit. Hence the result, crippled perception by the potential customer, that "open" is the synonym for "free of charge" or "cheapo". Open, used in the name of the software, becomes "unwelcome" out of marketing reasons.

Please tell me, why I should not promote the organization that pays my bills, is it a shame or what? As well as I would be happy to promote those donated to the project, is it bad? BTW, without one there would be no Aeroo - Alistek invested huge amount of development time to make it happen, fully paid to the developers.

**Comment by StefanRijnhart** [comment]
Feel free to publish and promote what and how you want. If it means not having Aeroo in the OCA then that is our loss, but it is your work. I don't see the problem myself. Being active in the community has given us a lot of promotion in itself even if the modules do not carry the Therp logo.
